src/server/http/__tests__/xsrf.js

```diff
@@ -104,14 +144,58 @@ describe('xsrf request filter', function () {
         expect(resp.payload).to.be('ok');
       });
 
-      it('rejects requests without either an xsrf or version header', async function () {
+      it('accepts requests with any allowed media type', async function () {
+        const allowedContentTypes = [
```

Comment:
In logs I see client requests with and without `charset`, likely `jQuery` and `Angular` use a bit different `content-type` string. So decided to cover this case in tests.